commitlint.config.js

```diff
@@ -1,3 +1,8 @@
+/**
+ *
+ * Copyright © 2025 Ping Identity Corporation
+ *
+ **/
```

Comment:
Are we sure this is the only text we need? The other platforms have a bit more to this copyright message: https://github.com/ForgeRock/ping-android-sdk/blob/develop/davinci/src/main/AndroidManifest.xml#L2

Reply:
I followed the google drive document https://drive.google.com/file/d/1y4tn5lzd7el5hsOKsa06moXr_1Bg0-8e/view and when I spoke with legal it seemed this was sufficient.